docs/mkdocs.yml

```diff
@@ -69,6 +70,7 @@ nav:
   - Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
   - Kafka Connect: kafka-connect.md
   - Integrations:
+    - amoro.md
```

Comment:
I don't think this is the right place, but it should replace the link above, like Daft and RisingWave.

Reply:
Thanks for correcting the document location